content/docs/connectors.md

```diff
@@ -0,0 +1,634 @@
+---
+draft: false
+title: "Syndesis connectors development"
+sidebar: "sidenav"
+menu:
+  sidenav:
+    name: Connectors development
+    weight: 3
+    parent: Developer Docs
+toc: true
+weight: 20
+---
+
+One of the most important features of Syndesis is to be able to connect to a variety of sources and destinations. The middleware developed is going to take care of transparently of the integration to such source/destination of data requiring a custom development of the specific transport involved in the chosen component.
```

Comment:
This phrase looks a bit complex for me. I would simplify it, something like:

"The developed middleware will take care transparently of the integration of such data source/destination. This requires a custom development of the specific transport involved in the chosen component."